x-pack/plugins/threat_intelligence/server/plugin_contract.ts

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
why not renaming this file to `types.ts` to be consistent with the frontend side of the plugin?